<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProxyList</class>
 <widget class="QWidget" name="ProxyList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XupaNet::ProxyList</string>
  </property>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>50</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Proxy List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSynchronizeProxies">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Sinchronize Proxies</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSetDefaultProxies">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>S&amp;et default Proxies</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>157</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Proxy 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditProxy1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Proxy 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditProxy2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Proxy 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditProxy3"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Proxy 4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditProxy4"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Proxy 5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditProxy5"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>81</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>port: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditPort1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditPort2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>port: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditPort3"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>port: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditPort4"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>port: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditPort5"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
